docs/basic_training/channels.pt.md

```diff
@@ -0,0 +1,696 @@
+| maxDepth      | Número máximo de níveis de diretório a serem visitados (padrão: <code>no limit</code>)                                                                       |
+| followLinks   | Quando `true` links simbólicos são seguidos durante a travessia da árvore de diretórios, caso contrário, eles são gerenciados como arquivos (padrão: `true`) |
+| size          | Define o número de arquivos que cada item emitido deve conter (padrão: 2). Use `-1` para qualquer número.                                                    |
+| flat          | Quando `true` os arquivos correspondentes são produzidos como únicos elementos nas tuplas emitidas (padrão: `false`).                                        |
+| checkIfExists | Quando `true`, lança uma exceção do caminho especificado que não existe no sistema de arquivos (padrão: `false`)                                             |
+
+!!! exercise
+
+    Use o método `fromFilePairs` para criar um canal emitindo todos os pares de leituras em fastq no diretório `data/ggal/` e imprima-os. Em seguida, use a opção `flat:true` e compare a saída com a execução anterior.
+
+    ??? solution
+
+        Use o seguinte, com ou sem `flat:true`:
+
+        ```groovy linenums="1"
+        Channel.fromFilePairs( './data/ggal/*_{1,2}.fq', flat:true)
+          .view()
+        ```
+
+        Em seguida, verifique os colchetes ao redor dos nomes dos arquivos, para ver a diferença com `flat`.
+
+### `fromSRA()`
+
+O método `Channel.fromSRA` permite consultar o banco de dados [NCBI SRA](https://www.ncbi.nlm.nih.gov/sra) e retorna um canal que emite os arquivos FASTQ correspondentes aos critérios de seleção especificados.
+
+A consulta pode ser ID(s) de projeto(s) ou número(s) de acesso suportado(s) pela API do [NCBI ESearch](https://www.ncbi.nlm.nih.gov/books/NBK25499/#chapter4.ESearch).
+
+!!! info
+
+    Esta função agora requer uma chave de API que você só pode obter fazendo login em sua conta NCBI.
+
+??? example "Instruções para login do NCBI e aquisição de chave"
+
+    1. Vá para: <https://www.ncbi.nlm.nih.gov/>
+    2. Clique no botão "Login" no canto superior direito para entrar no NCBI. Siga suas instruções.
+    3. Uma vez em sua conta, clique no botão no canto superior direito, geralmente seu ID.
+    4. Vá para Account settings
+    5. Role para baixo até a seção API Key Management.
+    6. Clique em "Criar uma chave de API".
+    7. A página será atualizada e a chave será exibida onde estava o botão. Copie sua chave.
+
+Por exemplo, o trecho a seguir imprimirá o conteúdo de um ID de projeto NCBI:
+
+```groovy linenums="1"
+params.ncbi_api_key = '<Your API key here>'
+
+Channel
+  .fromSRA(['SRP073307'], apiKey: params.ncbi_api_key)
+  .view()
+```
+
+!!! info ""
+
+    :material-lightbulb: Substitua `<Your API key here>` com sua chave de API.
```

Comment:
Se os outros elementos de código foram traduzidos em outras seções (por ex., em operadores), não deveríamos traduzir isso como:

```suggestion
    :material-lightbulb: Substitua `<Sua chave de API aqui>` com sua chave de API.
```

Reply:
Oh! True, I missed that. Thanks! 🥳